jpegenc/subblocks/quantizer/divider.py

```diff
@@ -0,0 +1,129 @@
+"""This module contains the HDL for
+    divider used for Quantiser"""
+
+from myhdl import always_seq, block, intbv, always_comb, Signal
+from .romr import romr
+
+
+def divider_ref(dividend, divisor):
```

Comment:
This is for testing only?  It might make more logical sense for this to be organized with the test code or in the models?